### PR TITLE
docs: update verbiage for licensing

### PR DESCRIPTION
**Pull request by stevenGravy**
The verbiage was a bit awkward in that this is not the only feature that would stop.

### CODE REVIEW

File: docs/pages/admin-guides/deploy-a-cluster/license.mdx [hunk 1]
```diff
@@ -151,7 +151,8 @@ Teleport Web UI users will see an alert banner:
 
 ![License expired banner](../../../img/enterprise/license-expired.png)
 
-Attempts to authenticate to Teleport via Single Sign-On connectors will fail.
+Attempts to authenticate to Teleport via SAML/OIDC Single Sign-On connectors will fail
+for example as an Enterprise feature.
```
Comment (zmb3) [inline docs/pages/admin-guides/deploy-a-cluster/license.mdx:155]: This grammar looks incorrect to me.

How about something like: Enterprise features like Single Sign-On will fail.

Comment (stevenGravy) [reply docs/pages/admin-guides/deploy-a-cluster/license.mdx]: thanks @zmb3 pls see update

**Review by ptgott — APPROVED** [review]

File: docs/pages/admin-guides/deploy-a-cluster/license.mdx [hunk 2]
```diff
@@ -151,7 +151,8 @@ Teleport Web UI users will see an alert banner:
 
 ![License expired banner](../../../img/enterprise/license-expired.png)
 
-Attempts to authenticate to Teleport via Single Sign-On connectors will fail.
+Enterprise features like Single Sign-On authentication with SAML/OIDC
```
Comment (zmb3) [inline docs/pages/admin-guides/deploy-a-cluster/license.mdx:154]: I still think this is unnecessarily verbose.

General rule of thumb, if you can remove part of the sentence and it remains just as clear as it was before, then you probably didn't need the extra words.

_Enterprise features like Single Sign-On will fail_ is fine. We don't need to mention SAML/OIDC connectors.

Comment (stevenGravy) [reply docs/pages/admin-guides/deploy-a-cluster/license.mdx]: please see update

**Review by scottpgallagher — APPROVED** [review]
Approved.

**Pull request merged**